core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20201013_project_per_workspace.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="dvoet" id="INVALID_BILLING_ACCT">
+        <addColumn tableName="BILLING_PROJECT">
+            <column name="INVALID_BILLING_ACCT" type="boolean" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet logicalFilePath="dummy" author="dvoet" id="workspace_version">
+        <addColumn tableName="WORKSPACE">
+            <column name="WORKSPACE_VERSION" type="VARCHAR(10)" defaultValue="v1">
```

Comment:
What was the rationale behind making this a string rather than an int? (I could see int as a bit nicer because we could increment it more easily/have enforced formatting standards? Just curious rather than as a blocker.)

Reply:
I have just envisioned it as v1 and v2 but it does not really matter. This is also not a long term thing. I don't anticipate a v3, etc.